packages/orbit-components/src/ChoiceTile/__tests__/index.test.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+import * as mq from "../../hooks/useMediaQuery";
+import ChoiceTile from "..";
+
+describe("ChoiceTile", () => {
+  it("renders with correct properties", () => {
+    render(
+      <ChoiceTile title="title" description="description" label="label" icon="icon" type="radio">
+        Custom content
+      </ChoiceTile>,
+    );
+    expect(screen.getByText("title")).toBeInTheDocument();
+    expect(screen.getByText("description")).toBeInTheDocument();
+    expect(screen.getByText("label")).toBeInTheDocument();
+    expect(screen.getByText("icon")).toBeInTheDocument();
+    expect(screen.getByText("Custom content")).toBeInTheDocument();
+    expect(screen.getByRole("radio")).toBeInTheDocument();
+  });
+
+  it("can be checkbox", () => {
+    render(<ChoiceTile type="checkbox" />);
+    expect(screen.getByRole("checkbox")).toBeInTheDocument();
+  });
+
+  it("can be selected", () => {
+    render(<ChoiceTile type="radio" selected />);
+    expect(screen.getByRole("radio")).toBeChecked();
+  });
+
+  it("calls onClick", () => {
```

Comment:
add that to the first section `renders with correct properties`, no need to have a separate assertion for this